config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
   get "welcome/index"
   root to: "welcome#index"
+  resources :users
+  get 'auth/:provider/callback', to: 'sessions#create'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.